repo/packages/E/es-proxy/0/marathon.json

```diff
@@ -0,0 +1,33 @@
+{
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{ es-proxy.image }}:{{ es-proxy.version }}",
+      "network": "BRIDGE",
+      "portMappings": [
+        {
+          "containerPort": 5601,
+          "hostPort": 0,
+          "protocol": "tcp"
+        }
+      ]
+    }
+  },
+  "env": {
+    "ELASTICSEARCH_URL": "{{ es-proxy.es_url }}"
```

Comment:
is this actually used in this package? I am not seeing this defined in the config.json. When I run the package, the env is set to:

```
"env":{"ELASTICSEARCH_URL":""}
```